src/components/CardFloatButton.vue

```diff
@@ -100,6 +157,51 @@
       },
       sleep: function(ms) {
         return new Promise(resolve => setTimeout(resolve, ms));
+      },
+      insertUser: function(){
+        var full_day = ''
+        let UserTeamPayload = null
+        var currentDate = new Date()
+        var day = currentDate.getDate()
+        var month = currentDate.getMonth() + 1
+        var year = currentDate.getFullYear()
+
+        if (currentDate.getDate() < 10){
+          full_day = (year + "-" + month + "-0" + day);
+        }else{
+          full_day = (year + "-" + month + "-" + day);
+        }     
+
+        if(this.user_team.length > 0) {
+          for(let i = 0; i < this.user_team.length; i++) {
+              if(this.sprint_values[i].initialDate <= full_day){
+                this.strint_started = true
+              }
+              if(this.user_team[i].idUser == this.id_user){
+                this.user_exist = true;
+              }
+              if(this.user_team[i].isScrumMaster == true){
```

Comment:
Negar a variável em vez de comparar com true: EX: if (!this.user_team[i].isScrumMaster)